<launch>
  <!-- Node to run the point cloud cluster detection -->
  <node name="pointcloud_cluster_detection_node" pkg="pointcloud_cluster_detection" type="pointcloud_cluster_detection_node" output="screen">
    <param name="min_size_x" type="double" value="0.03"/> <!-- Adjust as needed -->
    <param name="min_size_y" type="double" value="0.8"/> <!-- Adjust as needed -->
    <param name="min_size_z" type="double" value="0.8"/> <!-- Adjust as needed -->
    <param name="tolerance" type="double" value="0.09"/>   <!-- Adjust as needed -->
  </node>

  <!-- Node to run the Python position plotter -->
  <!-- <node name="position_plotter_node" pkg="pointcloud_cluster_detection" type="position_plotter.py" output="screen">
  </node> -->
</launch>
